### PR TITLE
feat: implement string function of expr

### PR DESCRIPTION
**Pull request by xuyukeviki**

### CODE REVIEW

**Comment by ngjaying** [comment]
Thank you!. Is this part of explain statement implementation? Please add unit tests.

**Comment by xuyukeviki** [comment]
> Thank you!. Is this part of explain statement implementation? Please add unit tests.

yep, I will add unit tests later. This pr is aimed to provide string function of expr so that we can explain logical plan easily

**Review by Yisaer — APPROVED** [review]

File: pkg/ast/statement.go [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright 2021-2023 EMQ Technologies Co., Ltd.
```
Comment (Yisaer) [inline pkg/ast/statement.go:1]: This shouldn't be changed.

File: pkg/ast/expr.go [hunk 2]
```diff
@@ -26,6 +31,7 @@ type NameNode interface {
 type Expr interface {
 	Node
 	expr()
+	String() string
```
Comment (Yisaer) [inline pkg/ast/expr.go:34]: I think `String` is not clear to describe what this function does，plz also add the comment

**Pull request merged**